ml-agents/tests/yamato/scripts/run_gym.py

```diff
@@ -1,14 +1,18 @@
 import argparse
 
 from gym_unity.envs import UnityEnv
+from mlagents_envs.environment import UnityEnvironment
 
 
 def test_run_environment(env_name):
     """
     Run the gym test using the specified environment
     :param env_name: Name of the Unity environment binary to launch
     """
-    env = UnityEnv(env_name, worker_id=1, use_visual=False, no_graphics=True)
+    base_env = UnityEnvironment(
+        env_name, worker_id=1, no_graphics=True, args=["-logFile", "-"]
+    )
+    env = UnityEnv(base_env, use_visual=False)
```

Comment:
One thing I don't like about this approach is it's not obvious whose responsibility it is to close the BaseEnv/UnityEnvironment (and UnityEnvironment.close() doesn't appear to be idempotent).